### PR TITLE
One

### PR DESCRIPTION
**Pull request by oralag2**
- [DEMO LINK](https://oralag2.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://oralag2.github.io/layout_search-bar-airbnb/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [ ] Icon implemented using background-image CSS property
- [ ] Inputs are written inside of 'form' tag with correctly passed attributes
- [ ] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [ ] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="https://www.google.com/search"
+          method="get"
+          class="container container__big"
+          data-qa="big"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:19]: Need to fix code style everywhere
```suggestion
    <form
      action="https://www.google.com/search"
      method="get"
      class="container container__big"
      data-qa="big"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,51 @@
...
+
+      <button type="submit"
+              class="search-button
+                     search-button--big">
```
Comment (volodymyr-soltys97) [inline src/index.html:23]: Redundant
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,102 @@
...
+.search__text--big {
+  font-size: 16px;
+}
+
+.search__text--small {
+  font-size: 14px;
+}
+
+.search__text::placeholder {
+  font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/style.css:70]: Default value, so you can just remove it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
The search field should span the entire width
image

**Comment by oralag2** [comment]
Help me I dont understand

**Pull request closed without merging**